custom/icds_reports/management/commands/push_pmo_data.py

```diff
@@ -0,0 +1,141 @@
+from django.core.management.base import BaseCommand
+import json
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
+from dateutil import parser
+from custom.icds_reports.models.views import PMOAPIView
+import hashlib
+from base64 import b64encode
+from Crypto.Cipher import AES
+from Crypto.Util.Padding import pad
+import requests
+
+
+class Command(BaseCommand):
+    help = "Update the SQL output files for ICDS Aggregation helpers"
+
+    # TODO: These following statics will be updated as NIC gives us the details
```

Comment:
what do they mean / can we link to where they are documented?

Reply:
yeah, sure. I have added asana ticket in which google doc is there. which tells that these details, NIC will be sending us and this data will be static for all requests we send to them. I think they use this info to identify from which server they are receiving data.